edgelight/edgelight.ino

```diff
@@ -274,35 +280,45 @@ void setup() {
   config.xclk_freq_hz = 20000000;
   config.pixel_format = PIXFORMAT_JPEG;
 
-  config.frame_size = FRAMESIZE_VGA; // other options include QVGA/CIF/VGA/SVGA/XGA/SXGA/UXGA
-  config.jpeg_quality = 20; // 20 at VGA should keep it under 16kb total in most cases
+  // Photo Quality Settings
+  config.frame_size = FRAMESIZE_UXGA; // See here for a list of options and resolutions: https://github.com/espressif/esp32-camera/blob/master/driver/include/sensor.h#L84
+  config.jpeg_quality = 10;           // lower means higher quality
```

Comment:
These lines contain the only real (non-formatting) changes

Reply:
there is some perf tradeoff here.  agreed that the default higher quality feels higher quality for first use though.  curious if we get better quality by scaling down the higher res image vs starting with the low-res image from the camera.  unclear how it generates the different resolution images internally.

Reply:
Agree we should tune this a bit more in the future. The previous settings made the captured images with the m5stack timer camera quite blurry, but we could try adjusting the quality back down as long as the images still look fine to human labelers.